crates/eframe/src/native/glow_integration.rs

```diff
@@ -986,7 +1008,7 @@ impl GlutinWindowContext {
                 class: ViewportClass::Root,
                 builder: viewport_builder,
                 info,
-                screenshot_requested: false,
+                actions_requested: HashSetExt::new(),
```

Comment:
I'm not sure what `HashSetExt` is, and I don't care to know

```suggestion
                actions_requested: Default::default(),
```